pact/ledger.pact

```diff
@@ -259,6 +266,7 @@
         { 'policy := policy:module{kip.token-policy-v1}
         , 'token := token }
         (policy::enforce-mint token account guard amount))
+      (emit-event (TRANSFER "" account amount))
```

Comment:
I'm somewhat uneasy that one can issue a `TRANSFER` event w/o passing the `TRANSFER` cap. That said, this PR doesn't have testing support yet so that may not actually be the case. Also, I do like the approach @ggobugi27 suggested (and as is found in `coin-v3`)